src/magento1/application/overlay/public/app/etc/local.xml.twig

```diff
@@ -77,7 +77,7 @@
         <routers>
             <adminhtml>
                 <args>
-                    <frontName><![CDATA[admin]]></frontName>
+                    <frontName><![CDATA[{{ @('admin.path') ?: 'admin' }}]]></frontName>
```

Comment:
We should probably add this to magento1's harness.yml so that it's easily visible that it can be modified. Perhaps under the `magento.admin.path` scope?

Reply:
👍